src/interfaces.ts

```diff
@@ -13,3 +14,21 @@ export interface Problem {
   instance?: string;
   [key: string]: any;
 }
+
+export const ALL_SPECS = Object.keys(specs);
+export const LAST_SPEC_VERSION = ALL_SPECS[ALL_SPECS.length - 1];
+
+export type SpecsEnum = keyof typeof specs | 'latest';
+
+export type ConvertRequestDto = {
+  /**
+   * Spec version to upgrade to.
+   * Default is 'latest'.
+   */
+  version?: SpecsEnum;
+  /**
+   * Language to convert the file to.
+   */
+  language?: string,
+  asyncapi: AsyncAPIDocument
+}
```

Comment:
We can move that DTO to the `convert.controller.ts` file, because I see that we only use it in that file.